src/api/__tests__/Feed.test.js

```diff
@@ -1367,31 +1398,81 @@ describe('api/Feed', () => {
 
         test('should throw if file does not have an id', () => {
             expect(() =>
-                feed.updateAnnotation({}, annotationId, text, permissions, successCallback, errorCallback),
+                feed.updateAnnotation({}, annotationId, text, undefined, permissions, successCallback, errorCallback),
             ).toThrow(fileError);
         });
 
+        test('should throw if no text or status', () => {
+            expect(() =>
+                feed.updateAnnotation(
+                    file,
+                    annotationId,
+                    undefined,
+                    undefined,
+                    permissions,
+                    successCallback,
+                    errorCallback,
+                ),
+            ).toThrowError();
+        });
+
         test('should set error callback and file', () => {
-            feed.updateAnnotation(file, annotationId, text, permissions, successCallback, errorCallback);
+            feed.updateAnnotation(file, annotationId, text, undefined, permissions, successCallback, errorCallback);
 
             expect(feed.errorCallback).toEqual(errorCallback);
             expect(feed.file).toEqual(file);
         });
 
-        test('should updateFeedItem with isPending set to true', () => {
-            feed.updateFeedItem = jest.fn();
-            feed.updateAnnotation(file, annotationId, text, permissions, successCallback, errorCallback);
+        describe('should updateFeedItem with isPending set to true', () => {
+            test.each`
+                testText     | testStatus   | expected                                                 | description
+                ${'hello'}   | ${undefined} | ${{ message: 'hello', isPending: true }}                 | ${'only text'}
```

Comment:
We generally try to avoid having the description or test summary in the test.each table. Should these just be separate tests?

Reply:
I don't think it makes sense to break it into separate tests as they would practically be copies. I can however remove the `description` column and modify the description of test to include the used values like so:
`'given text=$testText and status=$testStatus'`
What do you think?

Reply:
Sounds good.

Reply:
Done